rend3-types/src/lib.rs

```diff
@@ -646,25 +651,30 @@ pub enum CameraProjection {
     Orthographic {
         /// Size assumes the location is at the center of the camera area.
         size: Vec3A,
-        direction: Vec3A,
+        /// View matrix
+        view: Mat4,
```

Comment:
We should have the view matrix be part of the Camera directly.

Reply:
ah yeah, I was thinking about that at some point, but then concentrated on getting it to work first, that makes sense